src/components/Slider/Slider.module.css

```diff
@@ -0,0 +1,110 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # SLIDER
+\*------------------------------------*/
+
+/**
+ * Slider wrapping Label and Input
+ */
+.slider {
+  display: flex;
+  flex-direction: column;
+  gap: var(--eds-size-2-and-half);
+}
+
+/**
+ * Slider Input
+ */
+.slider__input {
+  --slider-track-height: var(--eds-size-1);
+  --slider-thumb-size: var(--eds-size-2-and-half);
+
+  appearance: none;
+  background: transparent;
+
+  height: var(--slider-thumb-size);
+}
+.slider__input:focus {
+  outline: none;
```

Comment:
This will focus the entire slider rather than the button

Reply:
Is it worth writing a `play` function that tabs onto the input so the focus is captured by chromatic or is that overkill?

Reply:
Ya, I think it's worth snapping!
Would it replace the default snap or should I make a new story for focus purpose?

Reply:
Found that the entirety of the slider is interactable, but made the hit box 44px high so it will be AAA compliant. Focus indicator remains on the thumb, however, and is snapped on chromatic